Source/missiles.cpp

```diff
@@ -198,7 +198,7 @@ static bool CheckBlock(Point from, Point to)
 	return false;
 }
 
-static inline bool InDungeonBounds(Point position)
+static inline bool InDungeonBounds(const Point position)

```

Comment:
Is there any improvement gained by declaring const when passing by value? I like using const reference for non-primitives, though I recall conversation on previous PRs where that was discouraged for relatively simple structs like Point.

Reply:
Yeah this seems unnecessary and also nonstandard as per the rest of the codebase. Would suggest removing.

Reply:
True, missed that it wasn't by reference.